src/MongoDB.Driver.Core/Core/WireProtocol/WriteWireProtocolBase.cs

```diff
@@ -160,7 +160,7 @@ private WriteConcernResult ProcessReply(ConnectionId connectionId, BsonDocument
 
             var response = reply.Documents.Single();
 
-            var notPrimaryOrNodeIsRecoveringException = ExceptionMapper.MapNotPrimaryOrNodeIsRecovering(connectionId, response, "err");
+            var notPrimaryOrNodeIsRecoveringException = ExceptionMapper.MapNotPrimaryOrNodeIsRecovering(connectionId, null, response, "err");
```

Comment:
Would it be wrong to pass the getLastErrorCommand?

Reply:
It would not be completely wrong. But it's not quite right either.

I considered faking up a synthetic delete, insert or update command but decided it was not worth it.

null means "I don't have access to the command", or "there was no command".

{ getLastError : 1 } isn't the *real* command. The real command (if it had existed) would have been a delete, insert or update command.

Given that information what's your recommendation? null or getlastError?

Reply:
It's about the expectation of a user of MongoCommandException.  The Command property doesn't say whether it can be null, and there are inline comments in the constructor that say it can be, but doesn't say it what circumstance.

I'm inclined more towards getLastError, but leave it to you.

Reply:
Pushed a new commit using getLastError instead of null.